src/test/java/de/zalando/aruha/nakadi/validation/EventBodyMustRespectSchema.java

```diff
@@ -46,7 +46,7 @@ public EventValidator materialize(final EventType eventType, final ValidationStr
         mapper.registerModule(new JsonOrgModule());
 
         final Function<OverrideDefinition, OverrideDefinition> enhanceWithQualifiedSchema = definition -> {
-            final JSONObject schema = eventType.getEventTypeSchema().getSchema();
+            final JSONObject schema = new JSONObject(eventType.getEventTypeSchema().getSchema());
```

Comment:
This was required since I changed the type of schema be a String, in order to be compliant with the API.